_data/events/20200602-virtual-pipelines-training.yml

```
@@ -0,0 +1,7 @@
+name: Virtual Pipelines Training
+startdate: 2019-06-2
+enddate: 2019-06-4
+location: Virtual
+meetingurl: https://indico.cern.ch/event/904759/
+image: 
+description: This is the first attempt at a fully virtual HEP Software Foundation training event to help you learn a single tools that is becoming ubiquitous in our community - "CI/CD" (if you aren't sure what that means and how to use it, you'll probably want to join us).
```

I would use a multi line yaml here. The quotes inside might be confusing it (and multi line is nicer, anyway).
```suggestion
description: >-
    This is the first attempt at a fully virtual HEP Software Foundation training event
    to help you learn a single tools that is becoming ubiquitous in our community -
    "CI/CD" (if you aren't sure what that means and how to use it, you'll probably
    want to join us).
```